Makefile

```diff
@@ -9,7 +9,7 @@ export IMAGE_VERSION = v1.40.0
 export SIMPLE_VERSION = $(shell (test "$(shell git describe --tags)" = "$(shell git describe --tags --abbrev=0)" && echo $(shell git describe --tags)) || echo $(shell git describe --tags --abbrev=0)+git)
 export GIT_VERSION = $(shell git describe --dirty --tags --always)
 export GIT_COMMIT = $(shell git rev-parse HEAD)
-export K8S_VERSION = 1.32.0
+export K8S_VERSION = 1.33.0
```

Comment:
I notice that when we bump it the image is no longer added in the kind.
I am not sure why. :-(

Reply:
I'm getting even weirder results locally where OLM won't install, I'll look more at this today.